conf/deployment/aws/ipi_3az_rhcos_3m_4w.yaml

```diff
@@ -0,0 +1,18 @@
+---
+#  This configuration file is used for Metro DR cluster
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'ipi'
+  region: 'us-east-2'
+  worker_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+  master_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  worker_replicas: 4
+  master_replicas: 3
+  worker_instance_type: 'm5.4xlarge'
```

Comment:
are we testing Metro DR on AWS ?

Reply:
@prsurve we are going to discuss this on mtg in 50 mins. The thing is that create such deployment on vSphere is not that easy and work on such deployment will require resources which we do not have much. 

Such deployment implementation on vSphere is not possible in range of 1 month I guess and if this will work and allow us to test such scenarios easily for disconnecting 1 AZ from the other I would go with this approach which we can do in timeframe of 4.7 GA.

But let's discuss it on the mtg.